neutron/assetlist.json

```diff
@@ -1323,6 +1323,44 @@
         "website": "https://www.drop.money/",
         "twitter": "https://x.com/Dropdotmoney"
       }
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "factory/neutron129ukd5cwahcjkccujz87rjemjukff7jf6sau72qrhva677xgz9gs4m4jeq/uarena",
+          "exponent": 0,
+          "aliases": [
+            "uarena"
+          ]
+        },
+        {
+          "denom": "arena",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/neutron129ukd5cwahcjkccujz87rjemjukff7jf6sau72qrhva677xgz9gs4m4jeq/uarena",
+      "name": "Arena Token",
+      "display": "arena",
+      "symbol": "ARENA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/arena_dao.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/arena_dao.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/arena_dao.png",
```

Comment:
The png and svg don't seem to be visually identical. The orange color is different and only the png has shading.

Reply:
Thanks for spotting that! Will mark as ready once I get the correct version.